Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -607,10 +568,35 @@ function (FixerInterface $a, FixerInterface $b) {
             }
         }
 
+        $rulesSets = array();
+        foreach (RulesSet::create()->getSetDefinitionNames() as $setName) {
```

Comment:
Have a static first, then I try to test them, mock them, and remembered why I hate statics.